p2p_wallet/Scenes/Main/SendToken/Subscenes/ChooseTokenAndAmount/SendToken.ChooseTokenAndAmount.ViewController.swift

```diff
@@ -12,17 +12,14 @@ extension SendToken.ChooseTokenAndAmount {
     class ViewController: SendToken.BaseViewController {
         // MARK: - Dependencies
         private let viewModel: SendTokenChooseTokenAndAmountViewModelType
```

Comment:
Should we inject view model here?

Reply:
Yes. Of course. This could be refactored later. I just refactored the main scenes, the subscenes weren't refactored. They will be refactored later